src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -110,17 +80,17 @@ public boolean equals(Object other) {
         }
 
         DeleteCommand otherDeleteCommand = (DeleteCommand) other;
-        return targetIndex.equals(otherDeleteCommand.targetIndex);
+        return targetIndexes.equals(otherDeleteCommand.targetIndexes);
```

Comment:
This implementation compares whether the arrays are identical. This will not work for two different arrays with same content. Use `Arrays.equals(targetIndex, otherDeleteCommand.targetIndexes)` instead.